src/commonMain/kotlin/exchange.dydx.abacus/output/Account.kt

```diff
@@ -846,6 +846,18 @@ data class SubaccountFillResources(
     }
 }
 
+@JsExport
+@Serializable
+enum class MarginMode(val rawValue: String) {
+    ISOLATED("ISOLATED"),
+    CROSS("CROSS");
+
+    companion object {
+        operator fun invoke(rawValue: String) =
+            MarginMode.values().firstOrNull { it.rawValue == rawValue }
+    }
+}
```

Comment:
@mike-dydx MarginMode exists in `TradeInput.kt`, should we remove it there and opt to use the `Account.kt` one everywhere? We should also wait to rebase over @johnqh 's changes